tests/jobs/test_scheduler_job.py

```diff
@@ -1059,6 +1067,112 @@ def test_process_file_should_failure_callback(self):
             os.remove(callback_file.name)
 
 
+class TestDagFileProcessorQueriesCount(unittest.TestCase):
```

Comment:
I am just wondering if we want to mark this test as `performance` and have a separate build for performance-only tests? In this way, it will be easier to spot PR / builds that change the performance.

Reply:
I think it's better to have it like it is now - > this way it will be executed for various DBs and pythons. And It is very fast

Reply:
Yes. These are very quick tests.  In the future, when we will work on tests that analyze complex cases, we can add markers and describe the principles on the basis of which we conclude that the test is a performance test. Then we will also have to think about the environment in which we want to run these tests and how often. These tests are very static tests and contain simple assertions for values.

Reply:
Performance tests are run, for example, daily, and then the results are collected into one central system to detect performance regression. They do not have identical results every time. Each time you run a test, you may have a different result.